release-notes/3.0/preview/api-diff/preview8/ASP.Net/3.0-preview8_Microsoft.AspNetCore.Mvc.ActionConstraints.md

```diff
@@ -0,0 +1,14 @@
+# Microsoft.AspNetCore.Mvc.ActionConstraints
+
+``` diff
+ namespace Microsoft.AspNetCore.Mvc.ActionConstraints {
++    public class HttpMethodActionConstraint : IActionConstraint, IActionConstraintMetadata {
++        public static readonly int HttpMethodConstraintOrder;
```

Comment:
@rynowak, I guess we should have left this non-public. Oh Well

Reply:
Why does this readonly field matter? We intentionally made this class public for API Versioning.